src/bin/pg_autoctl/file_utils.c

```diff
@@ -592,6 +592,73 @@ search_path(const char *filename, SearchPath *result)
 }
 
 
+/*
+ * search_path_deduplicate_symlinks traverse the SearchPath result obtained by
+ * calling the search_path() function and removes entries that are pointing to
+ * the same binary on-disk.
+ *
+ * In modern debian installations, for instance, we have /bin -> /usr/bin; and
+ * then we might find pg_config both in /bin/pg_config and /usr/bin/pg_config
+ * although it's only been installed once, and both are the same file.
+ *
+ * We use realpath() to deduplicate entries, and keep the entry that is not a
+ * symbolic link.
+ */
+bool
+search_path_deduplicate_symlinks(SearchPath *results, SearchPath *dedup)
+{
+	if (results->found < 2)
+	{
+		/* copy our results over to dedup and be done already */
+		*dedup = *results;
+		return true;
+	}
```

Comment:
I think it's nicer to remove this to get consistent behaviour. This way we'll always have the `realpath` of the binary in the output.

Reply:
Good point, done that way now.